actionkit/api/mailer.py

```diff
@@ -0,0 +1,150 @@
+import json
+import re
+import requests
+import sys
+
+from actionkit.api.base import ActionKitAPI
+
+class AKMailerAPI(ActionKitAPI):
```

Comment:
I think we should either rename the class and file to "mailing" (my preference, as the admin UI is all "mailing", and that seems more convention-setting than the API URLs) or rename the functions to "mailer." The mix of both seems like it will lead to confusion.